weathervane.config.docker

```diff
@@ -0,0 +1,60 @@
+{
+  # This is a temporary version of a docker config file.
+  # We can't use the fixed config sizes because the auctionBidServer does
+  # run in Docker-only yet.
+  "dockerNamespace" : "hrosenbe",
+  "dockerHosts" : [
+    { "name" : "AuctionDriver1" }, { "name" : "AuctionDriver2" }, 
+    { "name" : "AuctionDriver3" }, { "name" : "AuctionDriver4" }, 
+    { "name" : "AuctionDriver5" }, { "name" : "AuctionDriver6" }, 
+    { "name" : "AuctionDriver7" }, { "name" : "AuctionDriver8" }, 
+    { "name" : "AuctionApp1", }, { "name" : "AuctionApp2", }, { "name" : "AuctionApp3", }, 
+    { "name" : "AuctionBid1", }, 
+    { "name" : "AuctionWeb1", }, 
+    { "name" : "AuctionCs1", }, { "name" : "AuctionCs2", }, { "name" : "AuctionCs3", }, 
+    { "name" : "AuctionMsg1", }, 
+    { "name" : "AuctionDb1", }, 
+    { "name" : "AuctionNosql1", }, 
+  ],
```

Comment:
The reason that this can't be a simple list of Strings is that this is where a user would specify other attributes of a dockerHost.  For example:

"dockerHosts" : [  { "name" : "dockerHost1", "dockerPort" : 2375, "dockerNet" : "host", } ],

We need to be able to specify those attributes on a per-host basis